playbooks/roles/jumpbox/files/mongo_db_restore.sh

```diff
@@ -0,0 +1,72 @@
+#fyi: this is an unmodified fork from rex pilot. #todo: fix this
+
+#!/bin/bash
```

Comment:
fyi, please don't review this file as it remains unchanged from the rex pilot. jeff opened a story in our backlog to fix this file. for now, restoring the database will be accomplished manually via ssh and azure cli